.github/workflows/golangci-lint.yaml

```
@@ -0,0 +1,30 @@
+name: Lint & Annotate
```

```suggestion
name: Lint & Annotate for Golang
```

Or something like that.  Basically we have multiple languages so we should distinguish in the name that this is for the golang code.

---

Maybe "Lint & Annotate (Golang)"

---

done